src/models/ITestResultPayload.ts

```diff
@@ -26,4 +26,7 @@ export interface ITestResultPayload {
     testerName: string;
     testVersion?: string;
     reasonForCreation?: string;
+    contingencyTestNumber?: string
+    typeOfTest?: string;
```

Comment:
Suggestion: we already have `testType` - so instead of `typeOfTest` could we use `sourceType` ?
That way `source` and `sourceType` will sit next to each other and show they're related

Reply:
Although it would be a "nice to have" to have them next to each other, I'm not sure `sourceType` describes the intended purpose of the field. I'm not a big fan of both names to be honest, since `testTypes` was taken.

I can change if needed.

Reply:
I'll marked as closed since there are tickets dependent on this work, can open up a new PR to change it if needed